ideascube/monitoring/forms.py

```diff
@@ -112,11 +112,15 @@ class LoanForm(forms.ModelForm):
 
     def clean_specimen(self):
         barcode = self.cleaned_data['specimen']
-        if Loan.objects.due().filter(specimen__barcode=barcode).exists():
+        already_loaned_query = (Q(specimen__barcode=barcode)
+                               |Q(specimen__serial=barcode))
```

Comment:
~~Same remark as previously about how this might return 2 results.~~ Nope, this is fine here, you use it to test for existence, no problem.